contributingGuides/CONTRIBUTING.md

```diff
@@ -111,7 +111,7 @@ Additionally if you want to discuss an idea with the open source community witho
 3. If you cannot reproduce the problem, pause on this step and add a comment to the issue explaining where you are stuck or that you don't think the issue can be reproduced.
 
 #### Propose a solution for the job
-4. Do not propose solutions to jobs without the `Help Wanted` label applied. Any proposals submitted when that label is not present will not be reviewed.
+4. If you propose solutions to jobs without the Help Wanted label applied, know that the proposals will not be reviewed till the label is added and that we might never add the label, thus never hire a contributor
```

Comment:
How about: 
>You can propose solutions on any issue at at any time, but if you propose solutions to jobs before the `Help Wanted` label applied, you do so at your own risk. Proposals will not be reviewed until the label is added and there is always a chance that we might not add the label or hire an external contributor for the job.

Reply:
Sounds good, next time please use the suggest feature, so I can click to merge it 😄

Reply:
Yep yep. My b!